deployer/deployer-lib/deployer/app/appDeployer.go

```diff
@@ -0,0 +1,96 @@
+package app
+
+import (
+	"fmt"
+
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/config"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/connect"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/context"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/deployer/app/deployer"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/deployer/app/packager"
+	"github.com/RHEcosystemAppEng/SaaSi/deployer/deployer-lib/utils"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	err error
```

Comment:
same comment as before: can we really share the variable at package level?

Reply:
please view reply in previous thread